webrtc.html

```diff
@@ -7051,10 +7051,13 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </div>
       <p>The <code>failed</code> and <code>completed</code> states require an
       indication that there are no additional remote candidates. This can be
-      indicated either by <a data-link-for=
+      indicated by calling <code><a>addIceCandidate</a></code> with
+      a candidate value whose <code>candidate</code> property is set
+      to an empty string, by <a data-link-for=
       "RTCPeerConnection">canTrickleIceCandidates</a> being set to
-      <code>false</code>, or the processing of an end-of-candidates indication
-      as described in [[!JSEP]].</p>
+      <code>false</code>, or the processing of an end-of-candidates
+      indication as described
+      in <span data-jsep="addicecandidate">[[!JSEP]]</span>.</p>
```

Comment:
As I read JSEP at the moment, the end-of-candidates indication is the same as calling with a candidate value whose candidate property is the empty string, so this is saying the same thing twice.